Gemfile

```diff
@@ -1,8 +1,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
```

Comment:
Oh! No consternation on my end!! I'm not dogmatic about this stuff :).

Thanks for your willingness to review all these patches. I'll send 2-3 more
today and then Will calm down.
On Fri, May 8, 2015 at 9:18 AM Derrek Long <notifications@github.com> wrote:

> In Gemfile
> <https://github.com/derrek/street-address/pull/21#discussion_r29953253>:
>
> > @@ -1,8 +1,3 @@
> >  source 'https://rubygems.org'
> >
> >  gemspec
> > -
>
> Sorry didn't mean to cause any consternation. I noticed that you put them
> in the gemspec and wrote up a comment. Then I Fell asleep. Failed to press
> "Comment" button. Reviewing everything today.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/derrek/street-address/pull/21/files#r29953253>.
>